src/components/color-picker/color-picker-palette.tsx

```diff
@@ -0,0 +1,104 @@
+import { Component, h, State, Event, EventEmitter, Prop } from '@stencil/core';
+/**
+ * @private
+ */
+
+@Component({
+    tag: 'limel-color-picker-palette',
+    shadow: true,
+    styleUrl: 'color-picker-palette.scss',
+})
+export class PaletteExample {
+    /**
+     *
+     */
+    @Prop()
+    public searchedColor;
+
+    @State()
+    private value: string;
+
+    /**
+     * Emits chosen value to the parent component
+     */
+    @Event()
+    private change: EventEmitter<string>;
+
+    private colors = [
```

Comment:
May be move this to somewhere and can be reusable by other components if needed

Reply:
Sounds good. We have several examples in the Design Guidelines which use this list. But is the effort worth it? (also thinking if the PR gets too big).

Reply:
Moving it will not make the PR any bigger :smile: